src/app/page.tsx

```diff
@@ -1,9 +1,12 @@
 "use client";
-import React from "react";
+import React, { useState } from "react";
 import { useSearchParams, useRouter } from "next/navigation";
-import { useMetaIntegratedSystemsList } from "@/services/ecommerce/meta/hooks";
+import {
+  useMetaIntegratedSystemsList,
+  useMetaProductsList,
+} from "@/services/ecommerce/meta/hooks";
 import { styled } from "@mitodl/smoot-design";
-import { Typography } from "@mui/material";
+import { Typography, Button } from "@mui/material";
```

Comment:
We have a `Button` in smoot-design - this should use that so it matches the visual style of the rest of the app(s). (The props are pretty much the same between the two, I think, so this should be just swapping the imports. You can also pull in `styled` from smoot-design if you want to apply custom CSS to it too. See https://github.com/mitodl/unified-ecommerce-frontend/blob/main/src/page-components/CartSummary/CartSummary.tsx to see in action.)